static/templates/settings/account_settings.hbs

```diff
@@ -162,6 +162,10 @@
                       {{#unless user_can_change_avatar}}style="display:none"{{/unless}}>
                         &times;
                     </span>
+                    <span id="user_avatar_delete" aria-label="{{t 'Delete profile picture'}}" tabindex="0"
+                      {{#if (and (not page_params.is_admin) page_params.realm_avatar_changes_disabled)}}style="display:none"{{/if}}>
```

Comment:
What's your thinking on using this conditional here?  I would think it should just be unconditionally `display: none`?

Reply:
I thought when the admin activates `Prevent users from changing their avatar` in org permission settings. we should not show the text `Delete profile picture` text. and yes you are correct we wil not show the `X` icon when the admin activates
`Prevent users from changing their avatar` automaticaly hovering will never triggerd, I'll change and update soon